gcc/config/riscv/riscv-builtins.c

```diff
@@ -252,10 +252,13 @@ AVAIL (vector, TARGET_VECTOR)
 #define RISCV_ATYPE_C_SF_PTR const_float_ptr_type_node
 #define RISCV_ATYPE_C_DF_PTR const_double_ptr_type_node
 
+#define RISCV_ATYPE_VF16Mf4 rvvfloat16mf4_t_node
```

Comment:
Could you use all capital for marco name, that would be more consistency.

Reply:
I realize the point before, and some reasons here for this implementation,
the main purpose is that let 'f' as a fake number into current "gen-vector-iterator" design,
it becomes concise and smooth, although break consistency slightly :)

Reply:
I prefer to keep it keep it consistent, could you update that?

Reply:
Much codes use the same LMUL as FTYPE and function name,

For example:

#define _VINT_STRIDED_LOAD_STORE_BUILTINS(E, L, MLEN, MODE, SUBMODE, \
PNAME, PMODE, SUBTYPE, VCLASS)\
DIRECT_NAMED ( \
vlse##MODE##_##PNAME, vlse##SUBTYPE##E##m##L##_##PNAME, \
RISCV_##VCLASS##E##M##L##_FTYPE_C_##SUBMODE##_PTR_##PMODE, \
vector), \
DIRECT_NAMED (

The function name of builtins and FTYPE class all use the same “LMUL” item,

If we want to keep it consistent strictly, it will be more complex :(
Let ‘f’ as a part of number seems to be better way.

I do not find a elegant way to fix this by now, could you give me any advice to fix this ?